src/test/com/yetanalytics/lrs/pedestal/interceptor/xapi/statements/attachment_test.cljc

```diff
@@ -0,0 +1,139 @@
+(ns com.yetanalytics.lrs.pedestal.interceptor.xapi.statements.attachment-test
+  (:require [clojure.test :refer [deftest testing is] :include-macros true]
+            [clojure.spec.test.alpha :as stest :include-macros true]
+            [com.yetanalytics.test-support :refer [failures stc-opts]]
+            [com.yetanalytics.lrs.pedestal.interceptor.xapi.statements.attachment
+             :as attachment
+             :refer [decode-sig
+                     validate-sig
+                     validate-multiparts]])
+  #?(:clj (:import [java.io ByteArrayInputStream])))
+
+(deftest sig-test
+  (let [att-obj {"usageType" "http://adlnet.gov/expapi/attachments/signature",
```

Comment:
Might want to put these test fixtures into their own `def`s for code clarity.

Reply:
https://github.com/yetanalytics/lrs/pull/81/commits/437bf7bccdc78863846b5090a657b3fbd0f55b96

Reply:
Ah much better.